rolling-shutter/cmd/chain/init.go

```diff
@@ -170,24 +180,25 @@ func adjustPort(address string, keyperIndex int) (string, error) {
 
 func initFilesWithConfig(tendermintConfig *cfg.Config, config *Config, appState app.GenesisAppState) error {
 	var err error
-	// private validator
-	privValKeyFile := tendermintConfig.PrivValidatorKeyFile()
-	privValStateFile := tendermintConfig.PrivValidatorStateFile()
-	var pv *privval.FilePV
-	if tmos.FileExists(privValKeyFile) {
-		pv = privval.LoadFilePV(privValKeyFile, privValStateFile)
-		log.Info().
-			Str("privValKeyFile", privValKeyFile).
-			Str("stateFile", privValStateFile).
-			Msg("Found private validator")
-	} else {
-		pv = privval.GenFilePV(privValKeyFile, privValStateFile)
-		pv.Save()
-		log.Info().
-			Str("privValKeyFile", privValKeyFile).
-			Str("stateFile", privValStateFile).
-			Msg("Generated private validator")
-	}
+	if slices.Contains([]string{VALIDATOR, ISOLATEDVALIDATOR}, config.Role) {
```

Comment:
SEED role also needs privkey files

Reply:
Ehh, are you sure about that?

AFAICS the point of the seed role is to not be an actual part of the network but just to exchange peer information.

Reply:
No not sure, but IIRC the seed role node errored on me because it was looking for the file that was configured in the config.
Before this PR, the seed validator *did* get a privkey file.

So maybe it doesn't need it, but then this needs to be reflected in the tendermint config file as well.
Maybe the config file has to be the empty string then

Reply:
And apart from that - wouldn't it make sense that also seed nodes are authenticated by address /signature?

Reply:
They do need a node key (`node_key_file` in the config) for participating on the p2p network but what we're talking about here is the validator key.

I've just checked the boot nodes on the shapshot installation. They definitely don't have the `priv_*` key files.